velox/exec/tests/utils/TpchQueryBuilder.h

```diff
@@ -105,6 +106,11 @@ class TpchQueryBuilder {
   static const std::unordered_map<std::string, std::vector<std::string>>
       kTables_;
   static const std::vector<std::string> kTableNames_;
+
+  static const std::string kLineitem;
```

Comment:
These statics should be initialized here directly.

Reply:
This might likely cause a linter error described here in the internal Meta CI. 
https://github.com/facebookincubator/velox/pull/1155/files/602ff80ad3b5f7330fa07c09c366f8830a108c02#r822960183

Reply:
Try `static constexpr char* kLineitem = "lineitem";`